packages/cli/src/utils/config.js

```diff
@@ -74,5 +74,7 @@ export async function loadNuxtConfig(argv) {
     socket: argv['unix-socket'] || undefined
   }, options.server || {}, getDefaultNuxtConfig().server)
 
+  options.noLoadingScreen = !!argv['no-loading-screen']
```

Comment:
I will instead call is `loadingScreen` which is `true` by default :)

Reply:
I tried to do that but it didn't work. Help will still display `--no-loading-screen`

Reply:
What about `options.loadingScreen = !argv['no-loading-screen']`

Reply:
That might work.